pinot-core/src/test/java/org/apache/pinot/queries/InterSegmentOrderBySingleValueQueriesTest.java

```diff
@@ -165,7 +162,7 @@ public void testQueryOptions() {
         new Object[]{query, results, numDocsScanned, numEntriesScannedInFilter, numEntriesScannedPostFilter, numTotalDocs, dataSchema});
 
     // order by one of the group by columns DESC
-    query = "SELECT sum(column1) FROM testTable GROUP BY column11 ORDER BY column11 DESC";
+    query = "SELECT column11, sum(column1) FROM testTable GROUP BY column11 ORDER BY column11 DESC";
```

Comment:
Why is this change needed?

Reply:
this was using the pql endpoint with options because we were not ready for the sql parser. i changed it to use SQL endpoint. In sql endpoint if column is expecetd in results, then it needs to be in the selection

Reply:
We should not change existing unit tests. I suggest to append to the unit tests without changing the existing queries in existing tests otherwise we will end up reducing the coverage.

Reply:
We should simply add two unit tests -- one for PQL path and one for SQL path.

Reply:
there is a test for pql path in the same file. this one should always have been sql path test